packages/cli/src/WorkflowHelpers.ts

```diff
@@ -407,9 +409,10 @@ export async function saveStaticData(workflow: Workflow): Promise<void> {
 				// eslint-disable-next-line @typescript-eslint/no-use-before-define
 				await saveStaticDataById(workflow.id!, workflow.staticData);
 				workflow.staticData.__dataChanged = false;
-			} catch (e) {
+			} catch (error) {
+				if (error instanceof Error) ErrorReporterProxy.getInstance().error(error);
```

Comment:
Would it make sense to move the type guard to inside the call and handle the raw literal case in there to avoid having to add this check every time error is unknown, which will happen frequently?